tests/Engine/EngineBaseTest.php

```diff
@@ -109,6 +125,33 @@ public function testFilterValidLangsTesseractEngine(
 		}
 	}
 
+	/**
+	 * @param string[] $langs Language codes
+	 * @param string[] $validLangs
+	 * @param string $invalidLangsMode
+	 * @covers EngineBase::filterValidLangs for Transkribus Engine
+	 * @dataProvider provideTranskribusLangs
+	 */
+	public function testFilterValidLangsTranskribusEngine(
```

Comment:
This method looks very similar to the above `testFilterValidLangsTesseractEngine()`, and could perhaps be combined with that and only have one data provider method (which could provide an engine name as well). What do you think?

Reply:
Both methods have been Combined.